### ISSUE TITLE
Investigate first document publishing

### ISSUE THREAD

**Issue by noahmay** [Dsek-LTH/member-page#79]
## Short description of the feature
Need to be able to publish documents such as meeting protocols. 

Do this issue depend on another issue? Write Depends on #Issue_number here

[x] Relates to frontend
[x] Relates to the backend
[ ] Requires modification of GraphQL
[ ] Requires modification of Database
[x] Requires creating a new service

## Long description with technical specifications
A first deployment of this could just be upload to FTP server. Suggestions are welcome.
